tests/pages/thankyou.test.tsx

```diff
@@ -29,5 +36,27 @@ describe('pages', () => {
             const results = getServerSideProps(ctx);
             expect(results).toEqual(expectedResults);
         });
+
+        it('clears all session data except operator data', () => {
+            const operatorData: OperatorAttribute = {
+                name: 'Test Op',
+                nocCode: 'TEST',
+            };
+
+            const ctx = getMockContext({
+                cookies: {},
+                body: null,
+                session: {
+                    [OPERATOR_ATTRIBUTE]: operatorData,
+                },
+            });
+
+            getServerSideProps(ctx);
+
+            expect(ctx.req.session[OPERATOR_ATTRIBUTE]).toEqual(operatorData);
+            expect(ctx.req.session[FARE_TYPE_ATTRIBUTE]).toBeUndefined();
+            expect(ctx.req.session[SERVICE_ATTRIBUTE]).toBeUndefined();
+            expect(ctx.req.session[INPUT_METHOD_ATTRIBUTE]).toBeUndefined();
```

Comment:
Would it be worth setting these attributes at the same time as `operatorData` attribute is set, in order to confirm that they're cleared?